xonsh/tools.py

```diff
@@ -205,6 +216,28 @@ def __eq__(self, other):
         return all(map(operator.eq, self, other))
 
 
+class EnvPathDir(EnvPath):
```

Comment:
i don't think we need to subclass here, all the functionality below can be part of the EnvPath class above, PATH should always be a list of dirs anyway.